roles/runtime/tasks/crio.yaml

```diff
@@ -6,9 +6,9 @@
     state: present
     disable_gpg_check: true
 
-- name: Download CRI-O.
+- name: Download CRI-O - {{ crio_version }}
   unarchive:
-    src: "https://github.com/cri-o/cri-o/releases/download/v{{ crio_version }}/cri-o.ppc64le.v{{ crio_version }}.tar.gz"
+    src: "https://storage.googleapis.com/cri-o/artifacts/cri-o.ppc64le.v{{ crio_version }}.tar.gz"
```

Comment:
Assets are uploaded to storage.googleapis.com regardless of release, sometimes artefacts are not bundled with the release on GitHub, hence the change.